xonsh/base_shell.py

```diff
@@ -118,10 +117,10 @@ def default(self, line):
         src, code = self.push(line)
         if code is None:
             return
-        hist = builtins.__xonsh_history__
+        hist = builtins.__xonsh_history__  # pylint: disable=no-member
         ts1 = None
-        tee = Tee() if builtins.__xonsh_env__.get('XONSH_STORE_STDOUT') \
-                    else io.StringIO()
+        store_stdout = builtins.__xonsh_env__.get('XONSH_STORE_STDOUT', False)  # pylint: disable=no-member
```

Comment:
Please remove the `False` here.  The env's `get()` method provides its own defaults so that they can be specified in exactly one place in the code base.